.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [17, 21]
+        java: [17, 19, 21]
```

Comment:
Why are we adding Java 19?

Reply:
Don't we also need to run tests on version 19 to ensure the code is generated correctly?

Reply:
We are running the tests on Java 21, so that should cover this. Java 19 is no longer supported and we try to only run tests on LTS versions and the next upcoming version